.github/workflows/lint.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         working-directory: ${{ env.working_directory }}
         run: |
           sudo apt update && sudo apt install -y libgtest-dev libgmock-dev libbenchmark-dev &
-          pip install meson ninja &
+          pip install git+https://github.com/mesonbuild/meson ninja &
```

Comment:
Using nightly meson, because the latest release of meson doesn't know that gcc 11 supports `-std=c++23`.